tickets/tickets_commands.go

```diff
@@ -333,7 +333,7 @@ func (p *Plugin) AddCommands() {
 					CustomID: "tickets-open-" + reason,
 				}
 
-				if emoji != nil {
+				if emoji.ID != 0 || emoji.Name != "" {
```

Comment:
Won't this break non-custom emojis, due to their ID being "0" by default?

Reply:
The check is an OR, not an AND, and I think the name field should be set for default emojis.

Reply:
haha i n oticed before you